ArduCopter/Parameters.h

```diff
@@ -143,7 +143,7 @@ class Parameters {
         k_param_gpslock_limit,          // deprecated - remove
         k_param_geofence_limit,         // deprecated - remove
         k_param_altitude_limit,         // deprecated - remove
-        k_param_fence,
+        k_param_fence,                  // deprecated - remove
```

Comment:
```suggestion
        k_param_fence_old,                  // only used for conversion
```

lets the compiler help you on any usage - and we can't reuse/remove :-)